packages/components/src/Banner/components/BannerIcon/BannerIcon.css

```diff
@@ -5,6 +5,10 @@
   background-color: var(--color-surface--background);
 }
 
+:global(.jobber-retheme) .wrapper {
```

Comment:
When the text wraps this leaves it centered in the banner, doesn't it? (definitely the easiest but I think we want to keep it aligned to that first row of text)

Reply:
gonna be a weird one tho because 1st line of text is also aligned self to the middle until that button goes down below

Reply:
I know it's messy but is there a way that we could change the alignment at different sizes? I know we manipulate other elements in Banner based on component width. Something like

```
.wrapper.medium {
  align-self: center;
}
```

? Or would that not work because this is technically a subcomponent?

Reply:
it would work if i move all the CSS into 1 file haha